x/swagger.go

```diff
@@ -0,0 +1,19 @@
+//go:generate go run github.com/swaggo/swag/cmd/swag init --parseDependency -g swagger.go -o ../swagger
+
+package x
+
+import (
+	_ "github.com/cosmos/cosmos-sdk/types"
+)
+
+// @title CommercioNetwork API
+// @description Swagger API for CommercioNetwork
+// @contact.name Gianguido Sorà
```

Comment:
Maybe we should use something more generic for `@contact.name` and `@contact.email`.

@marcotradenet what could we use?

Reply:
I think the correct email should be developers@commercio.network